ui/src/pages/Home/ProjectsView/hooks/useGetProjects.tsx

```diff
@@ -0,0 +1,37 @@
+import {useEffect, useState} from "react";
+import {IProjectDetails} from "../../../../interfaces";
+import {dummyProjects} from "../../../../dummydata/dummy";
+import {DropResult, ResponderProvided} from "react-beautiful-dnd";
+
+export const useGetProjects = () => {
+    const [projects, setProjects] = useState<IProjectDetails[]>([]);
+    const retrieveProjects = async () => {
```

Comment:
I've notice that for constants you used CapitalCamelCase variables above. Here this is inconsistent because it dose not starts with Capital case

Reply:
Also the same naming inconsistency appears in the folders names, you have folders starting with capital case folders that dose not start with capital case. As a best practice I encourage you to use a style instead of both of them.

Reply:
yeah I was a bit undecided
like here i think:
https://alexmngn.medium.com/how-to-better-organize-your-react-applications-2fd3ea1920f1, where instead of scenes i use pages

Reply:
i like starting with small letter for folders and project structure and non React stuff
Because react components i like to have them CamelCase for react component folders i also use CamelCase

i do have an exception in components folder where now things are inconsistent
I was bounceing for multiple words between header-bar or headerBar or HeaderBar( to be the same as the component name)